src/main/java/com/gmail/inverseconduit/BotConfig.java

```diff
@@ -57,6 +62,21 @@ public BotConfig(Properties properties) {
         }
         this.rooms = Collections.unmodifiableList(rooms);
         LOGGER.log(Level.CONFIG, "Setting rooms to " + rooms);
+        
+      //---------------------------------------
+        value = properties.getProperty("ADMINS");
+        List<Integer> users = new ArrayList<>();
+        for (String v : value.split("\\s*,\\s*")) {
+        	try {
+        		Integer user = Integer.valueOf(v);
+        		users.add(user);
+        	} catch (NumberFormatException e) {
+        	    LOGGER.log(Level.CONFIG, "Skipping unparsable user ID.");
+                LOGGER.log(Level.FINEST, "", e);
```

Comment:
That was mostly copied from the parsing of the room numbers and with that I copied the log-statements. I am fairly new to this, so I thought they might be intended and with that they should be added to the block of reading privileged users as well.